google/cloud/storage/blob.py

```diff
@@ -3485,7 +3492,7 @@ def open(
             >>> client = storage.Client()
             >>> bucket = client.bucket("bucket-name")
 
-            >>> blob = bucket.blob("blob-name.txt")
+            >>> blob = bucket.get_blob("blob-name.txt")
```

Comment:
Suggestion: Add a comment stating:
```
bucket.get_blob() is retrieving object metadata which will pin generation to the latest value to prevent a race-condition during download.
```